packages/boxel-ui/addon/src/components/field-container/index.gts

```diff
@@ -106,6 +106,12 @@ const FieldContainer: TemplateOnlyComponent<Signature> = <template>
       flex: 1;
```

Comment:
I think it might be a better idea to get rid of this `flex: 1` here instead of adding the display none. We shouldn't be trying to style the contents of the yield. That's up to the consumer of this component.